src/migrator/sync.ts

```diff
@@ -0,0 +1,177 @@
+import {
+	modelText,
+	MigrationTuple,
+	MigrationError,
+	defaultMigrationCategory,
+	checkModelAlreadyExists,
+	setExecutedMigrations,
+	getExecutedMigrations,
+	lockMigrations,
+	MigrationCategories,
+	RunnableMigrations,
+} from './utils';
+import type { Tx } from '../database-layer/db';
+import type { Config, Model } from '../config-loader/config-loader';
+
+import * as _ from 'lodash';
+import * as sbvrUtils from '../sbvr-api/sbvr-utils';
+
+type ApiRootModel = Model & { apiRoot: string };
+
+export const postRun = async (tx: Tx, model: ApiRootModel): Promise<void> => {
+	const { initSql } = model;
+	if (initSql == null) {
+		return;
+	}
+
+	const modelName = model.apiRoot;
+
+	const exists = await checkModelAlreadyExists(tx, modelName);
+	if (!exists) {
+		(sbvrUtils.api.migrations?.logger.info ?? console.info)(
+			'First time executing, running init script',
+		);
+
+		await lockMigrations(tx, modelName, async () => {
+			try {
+				await tx.executeSql(initSql);
+			} catch (err) {
+				(sbvrUtils.api.migrations?.logger.error ?? console.error)(
+					`postRun locked sql execution error ${err} `,
+				);
+				throw new MigrationError(err);
+			}
+		});
+	}
+};
+
+export const run = async (tx: Tx, model: ApiRootModel): Promise<void> => {
+	const { migrations } = model;
+	if (
+		migrations == null ||
+		_.isEmpty(migrations) ||
+		migrations[defaultMigrationCategory] === undefined
```

Comment:
Pretty sure this will never run the old form of translations because for those `migrations[defaultMigrationCategory]` will always be undefined
```suggestion
		_.isEmpty(migrations)
```

Reply:
Yep, thx! deleting the defaultMigration check